src/explore-education-statistics-frontend/src/modules/subscriptions/NewSubscriptionPage.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { GetServerSideProps, NextPage } from 'next';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import Button from '@common/components/Button';
+import FormFieldTextInput from '@common/components/form/FormFieldTextInput';
+import FormProvider from '@common/components/form/FormProvider';
+import Form from '@common/components/form/Form';
+import useMounted from '@common/hooks/useMounted';
+import publicationService, {
+  PublicationTitle,
+} from '@common/services/publicationService';
+import Yup from '@common/validation/yup';
+import Page from '@frontend/components/Page';
+import notificationService from '@frontend/services/notificationService';
+import withAxiosHandler from '@frontend/middleware/ssr/withAxiosHandler';
+import SubscriptionStatusMessage from '@frontend/modules/subscriptions/components/SubscriptionStatusMessage';
+import Head from 'next/head';
+
+interface FormValues {
+  email: string;
+}
+
+interface Props {
+  slug: string;
+  data: PublicationTitle;
+}
+
+const SubscriptionPage: NextPage<Props> = ({ data, slug }) => {
+  const { isMounted } = useMounted();
+
+  if (!isMounted) {
+    return null;
+  }
```

Comment:
Could be me, but I don't understand why this is necessary

Reply:
Neither do I, but it was already there and I'm scared to touch it 🤣 I believe I've seen it in other files with a `<FormProvider>`, and I think it's something to do with the forms loading properly. @amyb-hiveit might know more?